ports/color-console/portfile.cmake

```
@@ -0,0 +1,16 @@
+vcpkg_from_github(
+    OUT_SOURCE_PATH SOURCE_PATH
+    REPO imfl/color-console
+    REF 745f57141e6322e20cdda59a74ab1e00d107ade1 #v1.0.0

```

```suggestion
    REF 745f57141e6322e20cdda59a74ab1e00d107ade1
```
This port does not have any release.